main.go

```diff
@@ -83,18 +85,55 @@ func (s *BookmarkService) AddBookmark(entry Entry) error {
 
 	resp, err := s.httpClient.Do(req)
 	if err != nil {
-		return fmt.Errorf("failed to send request: %w", err)
+		return "", fmt.Errorf("failed to send request: %w", err)
 	}
 	defer resp.Body.Close()
 
 	body, _ := io.ReadAll(resp.Body)
 
 	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusCreated {
-		return fmt.Errorf("unexpected status code: %d, body: %s", resp.StatusCode, string(body))
+		return "", fmt.Errorf("unexpected status code: %d, body: %s", resp.StatusCode, string(body))
+	}
+
+	var respData struct {
+		ID string `json:"id"`
+	}
+
+	if err := json.Unmarshal(body, &respData); err != nil {
+		return "", fmt.Errorf("failed to parse bookmark response: %v", err)
+		//fmt.Errorf("failed to parse bookmark response: %v", err)
 	}
 
 	log.Printf("Successfully created bookmark. Response: %s", string(body))
-	return nil
+	return respData.ID, nil
+}
+
+// https://docs.hoarder.app/api/add-a-bookmark-to-a-list
+func (s *BookmarkService) AddBookmarkToList(bookmarkID, listID string) error {
+    endpoint := fmt.Sprintf("%s/api/v1/lists/%s/bookmarks/%s", s.baseURL, listID, bookmarkID)
+    
+    req, err := http.NewRequest(http.MethodPut, endpoint, nil)
+    if err != nil {
+        return fmt.Errorf("failed to create request: %w", err)
+    }
+
+    req.Header.Set("Content-Type", "application/json")
+    req.Header.Set("Accept", "application/json")
+    req.Header.Set("Authorization", "Bearer "+s.apiToken)
+
+    resp, err := s.httpClient.Do(req)
+    if err != nil {
+        return fmt.Errorf("failed to send request: %w", err)
+    }
+    defer resp.Body.Close()
+
+    if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusCreated {
```

Comment:
As per the [Hoarder API docs](https://docs.hoarder.app/api/add-a-bookmark-to-a-list), this API returns only status 204 in case of success. Therefore, use this status code in the check:
```suggestion
    if resp.StatusCode != http.StatusNoContent {
```